src/Tools/ExternalAccess/NativeAOT/Internal/InterceptorUtils.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+internal static class InterceptorUtils
```

Comment:
`Internal` in an EA package is for backing code that the consuming location _shouldn't_ consume.